server-spi-private/src/main/java/org/keycloak/policy/ValidateOnLoginPasswordPolicyProvider.java

```diff
@@ -0,0 +1,26 @@
+package org.keycloak.policy;
+
+import org.keycloak.models.RealmModel;
+import org.keycloak.models.UserModel;
+
+public class ValidateOnLoginPasswordPolicyProvider implements PasswordPolicyProvider {
+    @Override
+    public PolicyError validate(RealmModel realm, UserModel user, String password) {
+        return null;
+    }
+
+    @Override
+    public PolicyError validate(String user, String password) {
+        return null;
+    }
+
+    @Override
+    public Object parseConfig(String value) {
+        return null;
+    }
+
+    @Override
+    public void close() {
```

Comment:
```suggestion
    public void close() {}
```

I personally like this style more, but I am not sure what the Keycloak style guidelines say.

Reply:
I looked at similar implementations and mainly found the current variant. Sometimes, the empty line is removed, like this:

```java
public void close() {
}
```